commit 35a3c891f6487ffafe2dc30ecf44f67fe51f166d
Author: Jonathan Neuschäfer <j.neuschaefer@gmx.net>
Date:   Sun Nov 8 14:24:14 2020 +0100

    docs: filesystems: link ubifs-authentication.rst without .rst extension
    
    Specifying the .rst extension doesn't cause any problems AFAICT, but
    it's uncommon.
    
    Signed-off-by: Jonathan Neuschäfer <j.neuschaefer@gmx.net>
    Link: https://lore.kernel.org/r/20201108132415.1789142-1-j.neuschaefer@gmx.net
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>

diff --git a/Documentation/filesystems/index.rst b/Documentation/filesystems/index.rst
index 98f59a864242..7be9b46d85d9 100644
--- a/Documentation/filesystems/index.rst
+++ b/Documentation/filesystems/index.rst
@@ -113,7 +113,7 @@ Documentation for filesystem implementations.
    sysv-fs
    tmpfs
    ubifs
-   ubifs-authentication.rst
+   ubifs-authentication
    udf
    virtiofs
    vfat